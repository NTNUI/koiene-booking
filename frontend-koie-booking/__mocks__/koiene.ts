import AdminBooking, { AdminBookingDictionary } from '../src/types/admin/AdminBooking';

export default function getKoieData(): AdminBookingDictionary {
  return koieData;
}

const koieData: AdminBookingDictionary = {
  ['flåkoia']: {
    name: 'Flåkoia',
    slug: 'flåkoia',
    numberOfBeds: 11,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 0,
      ['2020-09-16']: 0,
      ['2020-09-17']: 0,
      ['2020-09-18']: 11,
      ['2020-09-19']: 9,
      ['2020-09-20']: 0,
      ['2020-09-21']: 11,
      ['2020-09-22']: 11,
      ['2020-09-23']: 11,
      ['2020-09-24']: 11,
    },
  },
  ['mevasskoia']: {
    name: 'Mevasskoia',
    slug: 'mevasskoia',
    numberOfBeds: 5,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 5,
      ['2020-09-16']: 5,
      ['2020-09-17']: 5,
      ['2020-09-18']: 1,
      ['2020-09-19']: 0,
      ['2020-09-20']: 3,
      ['2020-09-21']: 5,
      ['2020-09-22']: 2,
      ['2020-09-23']: 5,
      ['2020-09-24']: 5,
    },
  },
  ['øvensenget']: {
    name: 'Øvensenget',
    slug: 'øvensenget',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['kåsen']: {
    name: 'Kåsen',
    slug: 'kåsen',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['stakkslettbua']: {
    name: 'Stakkslettbua',
    slug: 'stakkslettbua',
    numberOfBeds: 11,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 11,
      ['2020-09-16']: 11,
      ['2020-09-17']: 11,
      ['2020-09-18']: 11,
      ['2020-09-19']: 11,
      ['2020-09-20']: 11,
      ['2020-09-21']: 11,
      ['2020-09-22']: 11,
      ['2020-09-23']: 11,
      ['2020-09-24']: 11,
    },
  },
  ['holmsåkoia']: {
    name: 'Holmsåkoia',
    slug: 'holmsåkoia',
    numberOfBeds: 20,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 20,
      ['2020-09-16']: 20,
      ['2020-09-17']: 20,
      ['2020-09-18']: 20,
      ['2020-09-19']: 20,
      ['2020-09-20']: 20,
      ['2020-09-21']: 20,
      ['2020-09-22']: 20,
      ['2020-09-23']: 20,
      ['2020-09-24']: 20,
    },
  },
  ['sonvasskoia']: {
    name: 'Sonvasskoia',
    slug: 'sonvasskoia',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['nicokoia']: {
    name: 'Nicokoia',
    slug: 'nicokoia',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['telin']: {
    name: 'Telin',
    slug: 'telin',
    numberOfBeds: 9,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 9,
      ['2020-09-16']: 9,
      ['2020-09-17']: 9,
      ['2020-09-18']: 9,
      ['2020-09-19']: 9,
      ['2020-09-20']: 9,
      ['2020-09-21']: 9,
      ['2020-09-22']: 9,
      ['2020-09-23']: 9,
      ['2020-09-24']: 9,
    },
  },
  ['lyngli']: {
    name: 'Lyngli',
    slug: 'lyngli',
    numberOfBeds: 13,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 13,
      ['2020-09-16']: 13,
      ['2020-09-17']: 13,
      ['2020-09-18']: 13,
      ['2020-09-19']: 13,
      ['2020-09-20']: 13,
      ['2020-09-21']: 13,
      ['2020-09-22']: 13,
      ['2020-09-23']: 13,
      ['2020-09-24']: 13,
    },
  },
  ['kråklikåten']: {
    name: 'Kråklikåten',
    slug: 'kråklikåten',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['heinfjordstua']: {
    name: 'Heinfjordstua',
    slug: 'heinfjordstua',
    numberOfBeds: 25,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 25,
      ['2020-09-16']: 25,
      ['2020-09-17']: 25,
      ['2020-09-18']: 25,
      ['2020-09-19']: 25,
      ['2020-09-20']: 25,
      ['2020-09-21']: 25,
      ['2020-09-22']: 25,
      ['2020-09-23']: 25,
      ['2020-09-24']: 25,
    },
  },
  ['høgnabu']: {
    name: 'Høgnabu',
    slug: 'høgnabu',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['stabburet']: {
    name: 'Stabburet',
    slug: 'stabburet',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
  ['vekvessætra']: {
    name: 'Vekvessætra',
    slug: 'vekvessætra',
    numberOfBeds: 20,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 20,
      ['2020-09-16']: 20,
      ['2020-09-17']: 20,
      ['2020-09-18']: 20,
      ['2020-09-19']: 20,
      ['2020-09-20']: 20,
      ['2020-09-21']: 20,
      ['2020-09-22']: 20,
      ['2020-09-23']: 20,
      ['2020-09-24']: 20,
    },
  },
  ['fosenkoia']: {
    name: 'Fosenkoia',
    slug: 'fosenkoia',
    numberOfBeds: 10,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 10,
      ['2020-09-16']: 10,
      ['2020-09-17']: 10,
      ['2020-09-18']: 10,
      ['2020-09-19']: 10,
      ['2020-09-20']: 10,
      ['2020-09-21']: 10,
      ['2020-09-22']: 10,
      ['2020-09-23']: 10,
      ['2020-09-24']: 10,
    },
  },
  ['tågåbu']: {
    name: 'Tågåbu',
    slug: 'tågåbu',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['lynhøgen']: {
    name: 'Lynhøgen',
    slug: 'lynhøgen',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['selbukåten']: {
    name: 'Selbukåten',
    slug: 'selbukåten',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
  ['holvassgamma']: {
    name: 'Holvassgamma',
    slug: 'holvassgamma',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['kamtjønnkoia']: {
    name: 'Kamtjønnkoia',
    slug: 'kamtjønnkoia',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['rindalsløa']: {
    name: 'Rindalsløa',
    slug: 'rindalsløa',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['iglbu']: {
    name: 'Iglbu',
    slug: 'iglbu',
    numberOfBeds: 10,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 10,
      ['2020-09-16']: 10,
      ['2020-09-17']: 10,
      ['2020-09-18']: 10,
      ['2020-09-19']: 10,
      ['2020-09-20']: 10,
      ['2020-09-21']: 10,
      ['2020-09-22']: 10,
      ['2020-09-23']: 10,
      ['2020-09-24']: 10,
    },
  },
  ['mortenskåten']: {
    name: 'Mortenskåten',
    slug: 'mortenskåten',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
};
