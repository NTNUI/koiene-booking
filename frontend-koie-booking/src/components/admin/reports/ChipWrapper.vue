<template>
  <div @click="enableModal ? clickHandler : null">
    <StatusChip :chipText="ChipData.chipText" :chipColor="ChipData.chipColor" />
  </div>
</template>

<script lang="ts">
import Vue from 'vue';
import StatusChip from '@/components/admin/reports/StatusChip.vue';
import scssVars from '@/styles/variables.scss';

interface ChipData {
  chipText: string;
  chipColor: string;
}

export default Vue.extend({
  name: 'Reports',
  components: { StatusChip },
  props: {
    columnName: {
      type: String,
      default: '',
    },
    value: {
      type: Number,
      default: 0,
    },
    equipmentArray: {
      type: Array,
      default: null,
    },
  },
  computed: {
    columnIdentifier() {
      let color;
      let text;
      switch (this.columnName) {
        case 'gas':
          if (this.value === 0) {
            color = scssVars.globalColorRedStrong;
            text = 'Tomt';
          } else {
            color = scssVars.globalColorGreenStrong;
            text = 'Fullt';
          }
          break;
        case 'logs':
          break;
        case 'firewood':
          break;
        case 'boat':
          break;
        case 'canoe':
          break;
        case 'lifevest':
          break;
        case 'smokeDetector':
          break;
        case 'equipment':
          break;
      }
      return '';
    },
    woodScale() {
      let chipText;
      let chipColor;
      switch (this.value) {
        case 0:
          chipText = 'Tomt';
          chipColor = scssVars.globalColorRedStrong;
          break;
        case 1:
          chipText = 'Lite';
          chipColor = scssVars.globalColorRedWeak;
          break;
        case 2:
          chipText = 'Greit';
          chipColor = scssVars.globalColorYellow;
          break;
        case 3:
          chipText = 'Endel';
          chipColor = scssVars.globalColorGreenWeak;
          break;
        case 4:
          chipText = 'Mye';
          chipColor = scssVars.globalColorGreenStrong;
          break;
      }
      return 'hei';
    },
    boatScale() {},
  },
});
</script>

<style scoped></style>
