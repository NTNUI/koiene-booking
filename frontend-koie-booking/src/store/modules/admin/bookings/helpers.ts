import AdminBooking, { APIAdminBooking } from '@/types/admin/AdminBooking';

export function convertAPIBookingToAdminBooking(APIBooking: APIAdminBooking): AdminBooking {
  return {
    name: APIBooking.name,
    bedsAvailableInDateRange: APIBooking.beds_available_in_date_range,
    numberOfBeds: APIBooking.number_of_beds,
    slug: APIBooking.slug,
  };
}
