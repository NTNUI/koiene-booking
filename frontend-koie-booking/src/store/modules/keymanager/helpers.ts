import { APIKeyDetail } from '@/types/keyManager/KeyDetail';
import KeyDetail from '@/types/keyManager/KeyDetail';
import KEY_STATUS from '@/types/keyManager/KeyStatus';

export function convertAPIKeyDetailToKeyDetail(apiKeyDetail: APIKeyDetail): KeyDetail {
  const keyStatus: KEY_STATUS = (<any>KEY_STATUS)[apiKeyDetail.key_status.toUpperCase()];
  return {
    email: apiKeyDetail.contact_email,
    koieName: apiKeyDetail.koie,
    startDate: apiKeyDetail.arrival_date,
    endDate: apiKeyDetail.departure_date,
    keyStatus: keyStatus,
    uuid: apiKeyDetail.uuid,
  };
}
