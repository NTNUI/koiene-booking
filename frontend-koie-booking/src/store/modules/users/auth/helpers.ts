export function isLoggedIn(token: string | null): boolean {
  return token ? token.length > 0 : false;
}
