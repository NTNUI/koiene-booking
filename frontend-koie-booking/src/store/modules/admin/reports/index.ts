import { AdminReportsState, RootState } from '@/store/types';
import { Module } from 'vuex';
import { getters } from '@/store/modules/admin/reports/getters';
import { actions } from '@/store/modules/admin/reports/actions';
import { mutations } from '@/store/modules/admin/reports/mutations';

export const state: AdminReportsState = {
  chosenCabin: '',
  cabins: {},
  expanded: false,
};

const namespaced = true;

export const adminReports: Module<AdminReportsState, RootState> = {
  namespaced,
  state,
  getters,
  actions,
  mutations,
};
