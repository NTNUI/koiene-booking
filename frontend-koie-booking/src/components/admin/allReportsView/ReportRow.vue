<template>
  <tr>
    <td v-for="header in reportTableHeaders" :key="header.value">
      <EquipmentColumnDisplayer
        v-if="header.value === 'equipment'"
        :not-ok="report.equipmentNotOk"
        :not-sure="report.equipmentNotSure"
        :ok="report.equipmentOk"
      />
      <template v-else-if="report[header.value] !== undefined">
        <StatusChip
          v-if="reportTableColumns[header.value]"
          :color="getCorrectColorAndText(header.value).color"
          :text="String(getCorrectColorAndText(header.value).text)"
        />
        <StringDisplayer v-else :text="report[header.value]" />
      </template>
      <StringDisplayer v-else text="Fant ikke data for dette punktet" />
    </td>
  </tr>
</template>

<script lang="ts">
import Vue from 'vue';
import EquipmentColumnDisplayer from '@/components/admin/allReportsView/EquipmentColumnDisplayer.vue';
import StatusChip from '@/components/admin/allReportsView/StatusChip.vue';
import StringDisplayer from '@/components/admin/allReportsView/StringDisplayer.vue';
import { ColorAndText } from '@/types/admin/ReportColumn';
import { reportTableColumns } from '@/components/admin/allReportsView/ReportTableColumns';
import { reportTableHeaders } from '@/components/admin/allReportsView/ReportTableHeaders';

export default Vue.extend({
  name: 'ReportRow',
  components: { EquipmentColumnDisplayer, StatusChip, StringDisplayer },
  props: {
    report: {
      type: Object as () => any,
      default() {
        return null;
      },
    },
  },
  data() {
    return {
      reportTableHeaders,
      reportTableColumns,
    };
  },
  methods: {
    getCorrectColorAndText(header: string): ColorAndText {
      let chipColorAndText: ColorAndText;
      const options = reportTableColumns[header].colorAndTextOptions;
      try {
        chipColorAndText = options[this.report[header] as number];
      } catch (err) {
        console.log(err);
        chipColorAndText = { color: '#8E24AA', text: 'Feil ved lesing' };
      }
      return chipColorAndText;
    },
  },
});
</script>

<style scoped></style>
