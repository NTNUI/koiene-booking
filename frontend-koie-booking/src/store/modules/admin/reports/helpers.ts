import { APIAdminBooking } from '@/types/admin/AdminBooking';
import { KoieNameSlug } from '@/types/admin/AdminReports';
import APIAdminReport from '@/types/admin/APIAdminReport';
import AdminReport from '@/types/admin/AdminReport';

export function convertAPIBookingToKoieNameSlug(APIBooking: APIAdminBooking): KoieNameSlug {
  return {
    name: APIBooking.name,
    slug: APIBooking.slug,
  };
}

export function convertAPIReportToAdminReport(APIReport: APIAdminReport): AdminReport {
  return {
    timestamp: APIReport.timestamp,
    id: APIReport.id,
    slug: APIReport.slug,
    tripDate: APIReport.trip_date,
    bookingId: APIReport.booking_id,
    gasIsFull: APIReport.gas_is_full ? 1 : 0,
    firewood: APIReport.firewood,
    choppedUpWood: APIReport.chopped_up_wood,
    boatStatus: APIReport.boat_status,
    canoeStatus: APIReport.canoe_status,
    lifeJacketsStatus: APIReport.life_jackets_status,
    smokeDetectorIsWorking: APIReport.smoke_detector_is_working ? 1 : 0,
    equipmentOk: APIReport.equipment_ok,
    equipmentNotOk: APIReport.equipment_not_ok,
    equipmentNotSure: APIReport.equipment_not_sure,
    otherFaults: APIReport.other_faults,
    feedback: APIReport.feedback,
  };
}
