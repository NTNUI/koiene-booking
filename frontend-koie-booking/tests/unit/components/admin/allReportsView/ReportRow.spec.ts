import Vue from 'vue';
import Vuetify from 'vuetify';
import { createWrapper } from '../../../utils';
import { Wrapper } from '@vue/test-utils';

import ReportRow from '@/components/admin/allReportsView/ReportRow.vue';
import { getReport } from '../../../__mocks__/reports';
import { convertAPIReportToAdminReport } from '@/store/modules/admin/reports/helpers';
import scssVars from '@/styles/variables.scss'
Vue.use(Vuetify);

describe('Component ReportRow', () => {
  let wrapper: Wrapper<any>;

  beforeEach(() => {
    const report = convertAPIReportToAdminReport(getReport());
    const wrapperOptions = {
      propsData: {
        report,
      },
    };
    wrapper = createWrapper(ReportRow, wrapperOptions);
  });

  it('Matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Gets correct color and text for chip', () => {
    const colorAndText = wrapper.vm.getCorrectColorAndText("gasIsFull")
    expect(colorAndText).toEqual({text: 'Tomt', color: scssVars.globalColorRedStrong})
  });

  it('Gets shows error chip for wrong input', () => {
    const colorAndText = wrapper.vm.getCorrectColorAndText("This input is wrong")
    expect(colorAndText).toEqual({text: 'Feil ved lesing', color: '#8E24AA'})
  });
});
