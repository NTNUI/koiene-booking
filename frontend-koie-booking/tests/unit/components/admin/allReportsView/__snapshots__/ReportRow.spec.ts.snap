// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ReportRow Matches snapshot 1`] = `
<tr>
  <td>
    <div style="cursor: inherit;">
      <div style="text-align: center; white-space: nowrap; overflow: hidden; text-overflow: ellipsis; max-width: 150px;">
        2020-10-10
      </div>
      <!---->
    </div>
  </td>
  <td>
    <div style="cursor: inherit;">
      <div style="text-align: center; white-space: nowrap; overflow: hidden; text-overflow: ellipsis; max-width: 150px;">
        flakoia
      </div>
      <!---->
    </div>
  </td>
  <td>
    <div style="cursor: inherit;">
      <div style="text-align: center; white-space: nowrap; overflow: hidden; text-overflow: ellipsis; max-width: 150px;">
        2020-08-10
      </div>
      <!---->
    </div>
  </td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Tomt</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Lite</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">En del</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Ok</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Defekt</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Mangler Ã¥rer</span></span></td>
  <td><span class="v-chip theme--light v-size--default tomt white--text"><span class="v-chip__content">Defekt</span></span></td>
  <td>
    <div>
      <div class="row justify-center"><span class="mx-1 v-chip v-chip--clickable v-chip--no-color theme--light v-size--default"><span class="v-chip__content">
      4
    </span></span> <span class="mx-1 v-chip v-chip--clickable v-chip--no-color theme--light v-size--default"><span class="v-chip__content">
      7
    </span></span> <span class="mx-1 v-chip v-chip--clickable v-chip--no-color theme--light v-size--default"><span class="v-chip__content">
      4
    </span></span></div>
      <div role="dialog" class="v-dialog__container">
        <!---->
      </div>
    </div>
  </td>
  <td>
    <div style="cursor: pointer;">
      <div style="text-align: center; white-space: nowrap; overflow: hidden; text-overflow: ellipsis; max-width: 150px;">
        The cabin was missing
      </div>
      <div role="dialog" class="v-dialog__container">
        <!---->
      </div>
    </div>
  </td>
  <td>
    <div style="cursor: pointer;">
      <div style="text-align: center; white-space: nowrap; overflow: hidden; text-overflow: ellipsis; max-width: 150px;">
        Next time make sure the cabin is there
      </div>
      <div role="dialog" class="v-dialog__container">
        <!---->
      </div>
    </div>
  </td>
</tr>
`;
